Specs/Data/Cesium3DTiles/Metadata/PropertyAttributesPointCloud/README.md

```diff
@@ -0,0 +1,15 @@
+# Property Attributes Point Cloud
+
+TODO
```

Comment:
Just stumbled over this: 

On the one hand, I'd **STRONGLY** suggest that each of the example data sets here _should_ have a `README` - preferably with information about what the data set contains, what it is used for, how it was created, etc.. 

On the other hand: The other `Specs/Data` files do not have a `README.md`. So one could also make a case for just omitting this README, "for consistency" ( 😕 )

In doubt, it could be updated with the latest one from the sample models. I added _some_ more details there. Maybe not with as many technical details as one could wish for, but providing some context.

Reply:
Hi @javagl, I added an abbreviated version of your [README from the 3d-tiles-samples repo](https://github.com/CesiumGS/3d-tiles-samples/tree/main/glTF/EXT_structural_metadata/PropertyAttributesPointCloud) for now. (I didn't want to have to include the screenshot in the CesiumJS repo). I also included a link to the full README.
Let me know if this approach makes sense.